database.sql

```diff
@@ -360,7 +367,7 @@ CREATE TABLE IF NOT EXISTS `glink` (
 	`zcid` int(11) NOT NULL DEFAULT 0,
 	`updated` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
 	 PRIMARY KEY(`id`),
-	 INDEX `cid_uid_gcid_zcid` (`cid`,`uid`,`gcid`,`zcid`),
+	 UNIQUE INDEX `cid_uid_gcid_zcid` (`cid`,`uid`,`gcid`,`zcid`),
```

Comment:
This file is just a dump of the data structure, @annando didn't write it by hand himself.